zoloto/marker.py

```diff
@@ -92,9 +94,7 @@ def as_dict(self):
             "pixel_corners": self.__pixel_corners.tolist(),
         }
         try:
-            marker_dict.update(
-                {"rvec": self._rvec.tolist(), "tvec": self._tvec.tolist()}
-            )
+            marker_dict.update({"rvec": self._rvec.tolist(), "tvec": list(self._tvec)})
```

Comment:
Why'd 1 change to `list` and 1 not? Surely change neither or both? (Ideally neither)

Reply:
This PR doesn't change the type of `rvec`, I suspect that will change when I resolve conflicts in a sec.

Reply:
`s/sec/several hours because I didn't click the button/`

Reply:
I still see no change here?

Reply:
I know, haven't had an opportunity to sort it yet.

Reply:
Sorted. 672c68f